Makefile

```
@@ -35,6 +35,11 @@ export PRINT_HELP_PYSCRIPT
 help:
 	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
+.PHONY: format
+format: $(VENV_DIR)  ## auto-format the code using relevant tools
+	make isort
+	make black
```

```suggestion
	make black
	make flake8
```
I always do all three...